include/zephyr/input/input.h

```diff
@@ -136,11 +136,11 @@ struct input_callback {
  * for the callback structure. Useful if multiple callbacks are used for the
  * same callback function.
  */
-#define INPUT_CALLBACK_DEFINE_NAMED(_dev, _callback, _user_data, name)         \
-	static const STRUCT_SECTION_ITERABLE(input_callback, name) = {         \
-		.dev = _dev,                                                   \
-		.callback = _callback,                                         \
-		.user_data = _user_data,                                       \
+#define INPUT_CALLBACK_DEFINE_NAMED(_dev, _callback, _user_data, name)                             \
+	static const STRUCT_SECTION_ITERABLE(input_callback, _input_callback__##name) = {          \
+		.dev = _dev,                                                                       \
+		.callback = _callback,                                                             \
+		.user_data = _user_data,                                                           \
```

Comment:
yeah so if you could manually format and only change the lines that are needed it'd be great, less diffs to mess with, this should be

```c
#define INPUT_CALLBACK_DEFINE_NAMED(_dev, _callback, _user_data, name)         \
        static const STRUCT_SECTION_ITERABLE(input_callback,                   \
                                             _input_callback__##name) = {      \
                .dev = _dev,                                                   \
                .callback = _callback,                                         \
                .user_data = _user_data,                                       \
        }
```